internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -550,6 +554,14 @@ func (s *BackendService) convertALBListener() (*template.ALBListener, error) {
 	}, nil
 }
 
+func (s *BackendService) convertGracePeriod() *int64 {
+	var gracePeriod *int64 = aws.Int64(60)
```

Comment:
We should probably reuse `manifest.DefaultGracePeriod`: https://www.danielmsullivan.com/pages/tutorial_workflow_3bestpractice.html#no-magic-numbers

Reply:
unresolve the issue as it has not been resolved yet. Feel free to resolve after pushing the commit.

Reply:
oops I made the change for LBWS and not for BS. I will do it for BS as well.